cmd/crank/beta/validate/manager_test.go

```diff
@@ -0,0 +1,176 @@
+/*
+Copyright 2024 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validate
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/google/go-containerregistry/pkg/v1/static"
+	"github.com/google/go-containerregistry/pkg/v1/types"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	conregv1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/spf13/afero"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/crossplane/crossplane-runtime/pkg/test"
+)
+
+var (
+	configDep2Yaml = []byte(`apiVersion: meta.pkg.crossplane.io/v1alpha1
```

Comment:
nit:
```suggestion
	configMetaYaml = []byte(`apiVersion: meta.pkg.crossplane.io/v1alpha1
```
can you also add a yaml for a configuration package?

Reply:
I can add a yaml for a configuration package for sure but what would be the difference between that and the second object inside the `extensions` arg which has `pkg.crossplane.io/v1alpha1` as apiVersion?

Reply:
Ohh I assumed, you're using this meta object as an input, not as an output. Crossplane packages cannot  depend on meta objects. Therefore, it's safe to replace this manifest with a configuration package to verify validate command can download the dependencies which are configurations.

Reply:
@ezgidemirel can you please check the changes made at https://github.com/crossplane/crossplane/pull/5815/commits/75380c02b12253f32c75cf96c2952564d7bc1df3 to manager_test.go? I tried to follow your comments but not %100 sure if they are OK.